src/lando/main/scm/git.py

```diff
@@ -163,7 +166,7 @@ def apply_patch(
             f_diff.flush()
 
             cmds = [
-                ["apply", f_diff.name],
+                ["apply", "--reject", f_diff.name],
```

Comment:
It would be good to add a comment or something in the commit message about this parameter, and how it relates to `.rej` files.